apps/web/src/views/Notifications/components/NotificationItem/NotificationItemImage.tsx

```diff
@@ -4,18 +4,17 @@ import { ASSET_CDN } from 'config/constants/endpoints'
 import Image from 'next/image'
 import { CHAIN_NAME_TO_CHAIN_ID } from 'views/Notifications/constants'
 import {
+  extractChainIdFromAPRNotification,
   extractChainIdFromMessage,
   extractPercentageFromString,
-  extractWordBeforeFullStop,
   getBadgeString,
-  hasSingleFarm,
 } from 'views/Notifications/utils/textHelpers'
 import AlertIcon from '../../../../../public/images/notifications/alert-icon.svg'
 
 export const getNotificationPairlogo = (title: string, message: string) => {
-  const isAprNotification = title.includes('APR')
-  const chainName = isAprNotification ? extractWordBeforeFullStop(message) : extractChainIdFromMessage(message)
-  const chainId = CHAIN_NAME_TO_CHAIN_ID[chainName === 'polygon_zkevm.' ? 'polygon_zkevm' : chainName]
+  const isAprNotification = title.toLowerCase().includes('apr')
```

Comment:
It's not a good practice to depend on some string literal

Reply:
for now this is a limititation on the wallet connect notify api, as they dont have support yet for metadata when sending notifications, so theres no other way to extract things like chainId etc. they are support for metadata but for now to get things like chain id i have to extract it usin rgex form the notification body

however for you xample above i chan chang this flag to be from the notification scope id instead